config/webpack.common.js

```diff
@@ -125,7 +125,7 @@ module.exports = function (options) {
        *
        * See: https://webpack.js.org/configuration/resolve/#resolve-modules
        */
-      // modules: [ path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')]
+      modules: [ helpers.root('src'), helpers.root('node_modules')]
```

Comment:
BTW I don't understand the comment the tests fail - CI is passing, the unit tests pass for me locally...